__tests__/index.js

```diff
@@ -17,65 +21,67 @@ const mountProgressiveImage = (renderFn) => {
   );
 };
 
-
-describe('react-progressive-image', () => {
-
+describe("react-progressive-image", () => {
   beforeEach(() => {
     global.Image = Image;
   });
 
-  it('exports a React component', () => {
-    expect(typeof ProgressiveImage).toBe('function');
+  it("exports a React component", () => {
+    expect(typeof ProgressiveImage).toBe("function");
   });
 
-  it('throws if not provided a function as a child', () => {
+  it("throws if not provided a function as a child", () => {
+    /* eslint-disable no-console */
+    const _error = console.error;
+    console.error = jest.fn(() => {});
     expect(() => {
       mount(
         <ProgressiveImage>
           <h1>Uh oh!</h1>
         </ProgressiveImage>
       );
     }).toThrow(`ProgressiveImage requires a function as its only child`);
+    console.error = _error;
```

Comment:
If the exception fails, you've permanently mutate `console.error` after this test exits. Should refactor in a try/catch or restore in and `afterEach`